contents/handbook/company/team.mdx

```diff
@@ -540,90 +540,90 @@ In terms of cycling– a friend once convinced me to go on a bike ride from San
+  <a href="https://github.com/vania-pooh"><img src="https://avatars.githubusercontent.com/u/829320?v=4" title="vania-pooh" width="50" height="50"/></a>
+  <a href="https://github.com/irespaldiza"><img src="https://avatars.githubusercontent.com/u/11633327?v=4" title="irespaldiza" width="50" height="50"/></a>
+  <a href="https://github.com/croomes"><img src="https://avatars.githubusercontent.com/u/211994?v=4" title="croomes" width="50" height="50"/></a>
+  <a href="https://github.com/snormore"><img src="https://avatars.githubusercontent.com/u/182290?v=4" title="snormore" width="50" height="50"/></a>
+  <a href="https://github.com/faik"><img src="https://avatars.githubusercontent.com/u/43129?v=4" title="faik" width="50" height="50"/></a>
+  <a href="https://github.com/aandryashin"><img src="https://avatars.githubusercontent.com/u/1412461?v=4" title="aandryashin" width="50" height="50"/></a>
+  <a href="https://github.com/andrewsomething"><img src="https://avatars.githubusercontent.com/u/46943?v=4" title="andrewsomething" width="50" height="50"/></a>
+  <a href="https://github.com/Ferroin"><img src="https://avatars.githubusercontent.com/u/905151?v=4" title="Ferroin" width="50" height="50"/></a>
+  <a href="https://github.com/cpanato"><img src="https://avatars.githubusercontent.com/u/4115580?v=4" title="cpanato" width="50" height="50"/></a>
+  <a href="https://github.com/cakrit"><img src="https://avatars.githubusercontent.com/u/43294513?v=4" title="cakrit" width="50" height="50"/></a>
+  <a href="https://github.com/dkhenry"><img src="https://avatars.githubusercontent.com/u/489643?v=4" title="dkhenry" width="50" height="50"/></a>
+  <a href="https://github.com/oxplot"><img src="https://avatars.githubusercontent.com/u/483682?v=4" title="oxplot" width="50" height="50"/></a>
+  <a href="https://github.com/marc-barry"><img src="https://avatars.githubusercontent.com/u/4965634?v=4" title="marc-barry" width="50" height="50"/></a>
+  <a href="https://github.com/moabu"><img src="https://avatars.githubusercontent.com/u/47318409?v=4" title="moabu" width="50" height="50"/></a>
+  <a href="https://github.com/nawazdhandala"><img src="https://avatars.githubusercontent.com/u/2697338?v=4" title="nawazdhandala" width="50" height="50"/></a>
+  <a href="https://github.com/dar-mehta"><img src="https://avatars.githubusercontent.com/u/10489943?v=4" title="dar-mehta" width="50" height="50"/></a>
+  <a href="https://github.com/gmmorris"><img src="https://avatars.githubusercontent.com/u/386208?v=4" title="gmmorris" width="50" height="50"/></a>
+  <a href="https://github.com/bitdeli-chef"><img src="https://avatars.githubusercontent.com/u/3092978?v=4" title="bitdeli-chef" width="50" height="50"/></a>
+  <a href="https://github.com/nsidartha"><img src="https://avatars.githubusercontent.com/u/26918226?v=4" title="nsidartha" width="50" height="50"/></a>
```

Comment:
Given that these are self-closing now, could you then remove the closing `</a>` tags?

Reply:
What do you mean? I made the inner `<img>` self closing, and did not touch the outer `<a>...</a>`.

The diff without the attributes:

```diff
-  <a ...><img ...></a>
+  <a ...><img .../></a>
```

If I remove the closing `</a>`, then the link/`<a>` will no longer be closed?

```diff
-  <a ...><img ...></a>
+  <a ...><img .../>
```

Reply:
hahah sorry I completely missed the `img` tag, ignore me